tests/test_algorithm.py

```diff
@@ -2018,3 +2022,99 @@ def test_remove_data(self):
         # initially only data for X should be sent and on the last day only
         # data for Y should be sent since X is expired
         np.testing.assert_array_equal(self.algo.data, expected_lengths)
+
+
+class TestAssetAutoClose(TestCase):
+    """
+    Tests if delisted securities are properly removed from a portfolio holding
+    positions in said securities.
+    """
+    def setUp(self):
+        dt = pd.Timestamp('2015-01-05', tz='UTC')
+        self.env = TradingEnvironment()
+        ix = self.env.trading_days.get_loc(dt)
+
+        self.metadata = make_rotating_equity_info(
+            3, dt, trading_day, 2, 2,
+        )
+
+        # Make all start dates equal.
+        self.metadata['start_date'] = [self.metadata['start_date'][0]]*3
```

Comment:
You can just do `df[column] = scalar` instead of writing a list:

```
In [1]: df
Out[1]:
           a         b   c
0  -0.232151 -0.601906   5
1   1.795220 -0.535771   5
2   0.752929 -1.103029   5
3  -0.026765 -1.379774   5

In [2]: df.c = 1

In [3]: df
Out[3]:
           a         b  c
0  -0.232151 -0.601906  1
1   1.795220 -0.535771  1
2   0.752929 -1.103029  1
3  -0.026765 -1.379774  1
4   0.064691  1.046385  1
```